handlers/playmobile/playmobile.go

```diff
@@ -0,0 +1,185 @@
+package playmobile
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"net/http"
+	"encoding/xml"
+	"strings"
+	"encoding/json"
+
+	"github.com/nyaruka/courier"
+	"github.com/nyaruka/courier/handlers"
+	"github.com/nyaruka/courier/utils"
+)
+
+const (
+	configShortCode  = "shortcode"
+	configUsername = "username"
+	configPassword = "password"
+)
+
+var (
+	maxMsgLength = 160
+	sendURL      = "http://91.204.239.42:8083/broker-api/send"
```

Comment:
No domain name?

Reply:
Added "base_url" in channel config to save URL.